packages/cel-spec/scripts/fetch-testdata.js

```diff
@@ -32,17 +32,28 @@ const testdataTextProto = extractFiles(
   /^cel-spec-[^/]+\/tests\/simple\/testdata\/([^/]+\.textproto)$/,
 );
 // Convert textproto to JSON with `buf convert`, using the local module "proto".
-const testdataJson = convertTestDataToJson(
+const testdataJsonFiles = convertTestDataToJson(
   testdataTextProto,
   "proto",
   "cel.expr.conformance.test.SimpleTestFile",
 );
-// Write as JSON array to a TypeScript file
-writeFileSync(
-  "src/testdata-json.ts",
+
+// Write to JSON
+const writes = testdataJsonFiles.map((file) =>
+  fs.writeFile(
+    `src/testdata/json/${file.name}.json`,
+    JSON.stringify(file, null, 2),
+  ),
+);
+
+for await (const _ of writes);
```

Comment:
for-await-of will await each promise in sequence, and ultimately put more work on Node.js than a simple for-of with `writeSync`.

You want Promise.all:

```suggestion
// Write to JSON
await Promise.all(testdataJsonFiles.map((file) =>
  fs.writeFile(
    `src/testdata/json/${file.name}.json`,
    JSON.stringify(file, null, 2),
  ),
));
```